tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "exclude": ["**/node_modules/**", "**/dist/**"],
```

Comment:
The typescript server in vscode kept dying on me in the UI Kit repo - this is an attempt to make TS more reliable.